### PR TITLE
 ci win

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

**Comment by mli** [comment]
can you please remove the changes of indent?

**Comment by yajiedesign** [comment]
@mli ok indent remove

File: Jenkinsfile [hunk 1]
```diff
@@ -183,13 +252,53 @@ stage('Unit Test') {
...
+     }
+   },
+   'Python2/3: GPU Win':{
+     node('windows') {
+       ws('workspace/ut-python-gpu') {
+         init_git_win()
+         unstash 'vc14_gpu'
+         bat '''rmdir /s/q pkg_vc14_gpu
+7z x -y vc14_gpu.7z'''
+         bat """xcopy C:\\mxnet\\data data /E /I /Y
```
Comment (mli) [inline Jenkinsfile]: why we need to copy data and model? because the download program doesn't work on windows?

Comment (yajiedesign) [inline Jenkinsfile]: yes.

**Comment by mli** [comment]
i'm going to merge this PR, but the risk it breaks the ci is high.

**Comment by mli** [comment]
gpu test is failed. i suggest to remove the training test, you can just reuse the current setting

https://github.com/dmlc/mxnet/blob/master/Jenkinsfile#L130

**Pull request merged**